Terminal.Gui/Core/ConsoleDriver.cs

```diff
@@ -82,6 +82,86 @@ public enum Color {
 		White
 	}
 
+	/// <summary>
+	/// 24bit color. Support translate it to 4bit(Windows) and 8bit(Linux) color
+	/// </summary>
+	public struct TrueColor {
+		/// <summary>
+		/// Red color component.
+		/// </summary>
+		public int Red { get; }
+		/// <summary>
+		/// Green color component.
+		/// </summary>
+		public int Green { get; }
+		/// <summary>
+		/// Blue color component.
+		/// </summary>
+		public int Blue { get; }
+
+		/// <summary>
+		/// Initializes a new instance of the <see cref="TrueColor"/> struct.
+		/// </summary>
+		/// <param name="red"></param>
+		/// <param name="green"></param>
+		/// <param name="blue"></param>
+		public TrueColor (int red, int green, int blue)
```

Comment:
Please apply correct source code formatting to all files in this PR (hit Ctrl-K-D in VS).